t/security.t

```diff
@@ -0,0 +1,61 @@
+use Mojo::Base -strict;
+use Test::Mojo;
+use Test::More;
+
+use Mojolicious::Lite;
+post '/protected' => sub {
+  my $c = shift->openapi->valid_input or return;
+  $c->reply->openapi(200 => {ok => 1});
+  },
+  'protected';
+
+plugin OpenAPI => {
+  url      => 'data://main/sec.json',
+  security => {
+    dummy => sub {
+      my ($c, $config, $next) = @_;
+      return $c->$next if $c->req->headers->authorization;
+      return $c->render(json => $config, status => 401);
```

Comment:
No, since if you don't call `render()` or `$next`, then mojo will just sit there and nothing gets rendered. Just like you expect from a standard non-blocking under route. I could possibly add `delay()`, but I don't want to because it adds complexity that doesn't add anything from the standard mojo behaviour.